components/suggestion/suggestion_box.jsx

```diff
@@ -132,6 +132,16 @@ export default class SuggestionBox extends React.Component {
          * Passes the wrapper reference for height calculation
          */
         wrapperHeight: PropTypes.number,
+
+        /**
+         * Allows parent to access received suggestions
+         */
+        onSuggestionsReceived: PropTypes.func,
+
+        /**
+         * Supress loading spinner when necessary
```

Comment:
`Suppress`

Reply:
🙈

Reply:
heh, i've been there before, mine is even committed 🤫